backend/src/components/studentFilters.js

```diff
@@ -424,6 +434,18 @@ function createEllFundingFilter(pValue) {
   return ellFundingList;
 }
 
+function createRefugeeFundingFilter(pValue) {
+  let refugeeFundingList = [];
+
+  if (pValue.toString() === 'true') {
+    refugeeFundingList.push({ key: 'sdcStudentValidationIssueEntities.validationIssueCode', value: 'REFUGEEINSEPTCOL', operation: FILTER_OPERATION.NOT_IN, valueType: VALUE_TYPE.STRING, condition: CONDITION.OR });
```

Comment:
If we want to return students that don't have validationIssueCodes for the NOT_IN then we should use NONE_IN instead